notebooks/official/automl/sdk_automl_text_entity_extraction_online.ipynb

```diff
@@ -33,19 +33,20 @@
         "\n",
```

Comment:
<p>Please use "Vertex AI SDK for Python" instead of Vertex SDK (see: https://github.com/googleapis/python-aiplatform#vertex-ai-sdk-for-python)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/329/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>